npm-debug.log

```diff
@@ -1,27 +0,0 @@
-0 info it worked if it ends with ok
```

Comment:
So this is the only conflict and since it's the npm-debug.log, go with what's working on your branch.